packages/arb-token-bridge-ui/src/hooks/__tests__/useSelectedTokenBalances.test.ts

```diff
@@ -6,9 +6,12 @@ import { getWagmiChain } from '../../util/wagmi/getWagmiChain'
 import { useNetworks } from '../useNetworks'
 import { useBalances } from '../useBalances'
 import { useSelectedTokenBalances } from '../TransferPanel/useSelectedTokenBalances'
-import { useAppState } from '../../state'
+import { useSelectedToken } from '../useSelectedToken'
 import { ChainId } from '../../types/ChainId'
 
+type BridgeToken = NonNullable<ReturnType<typeof useSelectedToken>[0]>
+const Erc20Type = 'ERC20' as BridgeToken['type']
```

Comment:
Both type seem to be `any`. I think it's related to recent changes with parsing the lists. Typescript is not able to infer what's the type of `data` anymore

Reply:
Also, I would use `satisfies` over `as` here. We still want typecheck to happen